pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -983,6 +994,72 @@ func setJvmDefault(jvm *api.JvmObject) {
 	}
 }
 
+func setResourcesDefaultServer(server *api.KieServerSet) { //resources *corev1.ResourceRequirements) {
+	if server.Resources != nil {
+
+		if server.Resources.Limits == nil {
+			server.Resources.Limits = createCPULimit(constants.ServersCPULimit)
+		}
+
+		if server.Resources.Requests == nil {
+			server.Resources.Requests = createCPURequests(constants.ServersCPURequests)
+		}
+	} else {
+		server.Resources = createResourceWithLimitsAndRequests(constants.ServersCPULimit, constants.ServersCPURequests)
+	}
+}
+
+func setResourcesDefaultConsole(console *v2.ConsoleObject) {
```

Comment:
instead of using a separate method for each object type... maybe instead we can have 1 method that all of them can use? Consider that KieAppObject is what actually houses the Resources struct and that KieAppObject exists in console, servers, and smartrouter. What if this method instead look like this? 
```
func setResourcesDefault(console *v2.KieAppObject, limits, requests string) {
```
Could console, servers and smartrouter all potentially use that same method?